### PR TITLE
fix/ remove `ManagedPoolController` from the repo

### PR DESCRIPTION
**Pull request by 0xArdi**
Removes the `ManagedPoolController` from the repo, as a result of https://github.com/balancer-labs/balancer-v2-monorepo/pull/2100

### CODE REVIEW

File: packages/balancer/contracts/managed_pool/contract.py [hunk 1]
```diff
@@ -112,3 +113,147 @@ def get_normalized_weights(
...
+    ) -> Dict[str, Any]:
+        """
+        Get the encoded params for a update weights gradually function call.
+
+        Note: This transaction is not being signed here. We simply encode the data for calling `updateWeightsGradually`
+        function. This data is later passed to the safe contract, that will ultimately make the tx.
+
+        :param must_allowlist_lps: The datetime from which the update should begin.
+        :return: the encoded tx data
+        """
```
Comment (0xArdi) [inline packages/balancer/contracts/managed_pool/contract.py:132]: This is wrong, will be updated on the appropriate PR.

**Comment by 0xArdi** [comment]
The `gitleaks` failure is a false positive. I will address it at one of the stacked PRs.

**Review by dvilelaf — APPROVED** [review]
LGTM. You just need to add some fingerprint to the leak ignore

File: packages/balancer/agents/autonomous_fund/tests/test_contracts/test_managed_pool/test_contract.py [hunk 2]
```diff
@@ -158,6 +128,10 @@ def test_get_gradual_weight_update_params(self) -> None:
             == INITIAL_POOL_WEIGHTS
         )
 
+    def teardown_class(self) -> None:
```
Comment (DavidMinarsch) [inline packages/balancer/agents/autonomous_fund/tests/test_contracts/test_managed_pool/test_contract.py:131]: Please check you're not overriding `super`'s `teardown_class` method but extend it

**Review by DavidMinarsch — APPROVED** [review]

File: packages/balancer/agents/autonomous_fund/tests/test_contracts/test_managed_pool/test_contract.py (hunk 2 above)
Comment (0xArdi) [reply packages/balancer/agents/autonomous_fund/tests/test_contracts/test_managed_pool/test_contract.py:131]: None of the super classes have `teardown_class`. None of them are test classes.

**Pull request merged**